### PR TITLE
make owe transition wireless profile the new default

### LINKED ISSUE #237: Implement owe transition
Openwrt supports now openwrt transition ifname on the same phy:
https://github.com/openwrt/openwrt/commit/574539ee2cdbb3dd54086423c6dfdd19bb1c06a6

OWE Transition:
https://posts.specterops.io/war-never-changes-attacks-against-wpa3s-enhanced-open-part-2-understanding-owe-90fdc29126a1

[ToDo: add better description]

### PR DESCRIPTION
**Pull request by spolack**
i've encountered zero issues, even the owe usage rate increased, because it is taken automatically when available.

fixes #237

### CODE REVIEW

**Comment by PolynomialDivision** [comment]
Works for me after deleting the "berlin.freifunk.net (Encrypted)" from my NetworkManager. Not sure why it was necessary (a reboot also did not help)

**Review by PolynomialDivision — APPROVED** [review]

**Comment by PolynomialDivision** [comment]
Runs fine since several days now. Great feature. :)

**Review by Noki — APPROVED** [review]
added it to my local node and works fine so far.

**Comment by PolynomialDivision** [comment]
What about merging this?

**Comment by Noki** [comment]
Works for several days here with my equipment. @spolack feel free to rebase and merge at any time.

**Comment by spolack** [comment]
@PolynomialDivision did you investigate the issues you experienced on Fritzbox 4040?

**Comment by spolack** [comment]
@Akira25 would you like to test aswell?

@pktpls i recall you recently had issues. Were they related to that change?

**Comment by PolynomialDivision** [comment]
> @PolynomialDivision did you investigate the issues you experienced on Fritzbox 4040?

Hmm, I was running snapshot without any issues, however switching back to 22.03 creates again some issues:
```
[ 6925.622409] wlp3s0: authenticate with 2a:3a:xx:xx:xx:b8
[ 6925.641528] wlp3s0: send auth to 2a:3a:xx:xx:xx:b8 (try 1/3)
[ 6925.649113] wlp3s0: authenticated
[ 6925.676225] wlp3s0: associate with 2a:3a:xx:xx:xx:b8 (try 1/3)
[ 6925.780354] wlp3s0: associate with 2a:3a:xx:xx:xx:b8 (try 2/3)
[ 6925.882922] wlp3s0: associate with 2a:3a:xx:xx:xx:b8 (try 3/3)
[ 6925.909408] wlp3s0: RX AssocResp from 2a:3a:xx:xx:xx:b8 (capab=0x11 status=0 aid=2)
[ 6925.912616] wlp3s0: associated
[ 6926.499272] iwlwifi 0000:03:00.0: No beacon heard and the time event is over already...
[ 6926.499311] wlp3s0: Connection to AP 2a:3a:xx:xx:xx:b8 lost
```
Sometimes it works, sometimes not. It was far better with snapshot.

File: group_vars/all/wireless_profiles.yml [hunk 1]
```diff
@@ -42,41 +42,6 @@ wireless_profiles:
...
-        encryption: owe
-        network: dhcp
-        radio: [11a_standard, 11g_standard]
-        ifname_hint: ffowe
-        ieee80211w: 1
-
-      - mode: mesh
-        mesh_id: Mesh-Freifunk-Berlin
-        radio: [11a_standard, 11g_standard, 11a_mesh]
-        mcast_rate: 12000                              # inzwischen 16000?
```
Comment (Akira25) [inline group_vars/all/wireless_profiles.yml:64]: Maybe we should have a look at the mcast-rate sometime in the future, but that is not a blocking issue.

**Review by Akira25 — APPROVED** [review]
Looks good to me from the code side.

**Pull request merged**